IBX-11150: Recommendation blocks described in Dev-doc (#3103)
ezsystems/developer-documentation: docs/cdp/raptor_integration/pb_blocks.md

---
description: Recommendation blocks in Page Builder
edition: experience
---

# Recommendation blocks in Page Builder

One of the Raptor Integration elements is the introduction of recommendation blocks available in the [Page Builder](page_builder_guide.md).

> Each Content, Product, or Commerce recommendation can be added to a landing page using the blocks.

Reviewer: ```suggestion
Content, Product, and Commerce recommendation can be added to a landing page using the blocks.
```

    - [Most popular products]([[= user_doc =]]/personalization/raptor_integration/raptor_recommendation_blocks/#most-popular-products-block)
    - [Most popular products in category]([[= user_doc =]]/personalization/raptor_integration/raptor_recommendation_blocks/#most-popular-products-in-category-block)
    - [Other customers have also seen]([[= user_doc =]]/personalization/raptor_integration/raptor_recommendation_blocks/#other-customers-have-also-seen-block)
- **Recommendations: Commerce** - shows recommendations based on visitors' purchase history (buy and basket events):
    - [The Personal Shopping Assistant]([[= user_doc =]]/personalization/raptor_integration/raptor_recommendation_blocks/#the-personal-shopping-assistant-block)
    - [User's item history or current basket items sorted by recent items or top items]([[= user_doc =]]/personalization/raptor_integration/raptor_recommendation_blocks/#users-item-history-or-current-basket-items-sorted-by-recent-items-or-top-items-block)

![Recommendation blocks](img/recommendation_blocks.png)

After opening the settings of a recommendation block, a link is available at the bottom of the window.

> It leads to the Raptor Control Panel (opens in a separate tab), where you can configure advanced settings and fine-tune the recommendation strategy.

Reviewer: I see https://github.com/ibexa/documentation-developer/pull/3101/changes is linking to the Control Panel, maybe we add a link here as well?
```suggestion
It leads to the [Raptor Control Panel](https://controlpanel.raptorsmartadvisor.com/) (opens in a separate tab), where you can configure advanced settings and fine-tune the recommendation strategy.
```
